cypress/e2e/check-characters-words-typos.cy.js

```diff
@@ -0,0 +1,18 @@
+describe("its going to test if the characters change, words, typos change as we write", () => {
+    beforeEach(() => {
+        cy.visit("/");
+    });
+
+    it("will click on dokument and on shkruaj again", () => {
```

Comment:
Ideally we break this test up into different tests, but if you want to keep it like this maybe we can reword it to "will correctly update `karaktere, fjalë, shenjime`"?